### PR TITLE
added logical operations to :has()

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/css/_colon_has/index.md [hunk 1]
```diff
@@ -174,6 +174,19 @@ This selector could have also been written as:
 }
 ```
 
+## Logical operatations
```
Comment (teoli2003) [inline files/en-us/web/css/_colon_has/index.md:177]:
```suggestion
### Logical operations
```

File: files/en-us/web/css/_colon_has/index.md [hunk 2]
```diff
@@ -174,6 +174,19 @@ This selector could have also been written as:
...
+
+The `:has()` relational selector can be used to check if one of multiple features is true, or if all the features are true.
```
Comment (teoli2003) [inline files/en-us/web/css/_colon_has/index.md:179]:
```suggestion
The `:has()` relational selector can be used to check if one of the multiple features is true or if all the features are true.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few tiny changes. Also I moved the section as a H3 (under examples), or else it should be earlier in the page.

File: files/en-us/web/css/_colon_has/index.md [hunk 3]
```diff
@@ -174,6 +174,19 @@ This selector could have also been written as:
...
+
+The `:has()` relational selector can be used to check if one of multiple features is true, or if all the features are true. 
+
+By using comma-separated values inside the `:has()` relational selector, you are checking to see if any of the parameters exist. `x:has(a, b)` will style `x` if descendant `a` OR `b` exists.
+
+By chaining together multiple `:has()` relational selectors together, you are checking to see all of the parameters exist. `x:has(a):has(b)` will style `x` if descendant `a` AND `b` exist.
+
+```css
+body:has(video, audio) {/*styles to apply if the content contains audio OR video */ }
+body:has(video):has(audio) {/*styles to apply if the content contains both audio AND video */ }
```
Comment (Josh-Cena) [inline files/en-us/web/css/_colon_has/index.md:187]:
```suggestion
body:has(video, audio) {
  /* styles to apply if the content contains audio OR video */
}
body:has(video):has(audio) {
  /* styles to apply if the content contains both audio AND video */
}
```

File: files/en-us/web/css/_colon_has/index.md [hunk 4]
```diff
@@ -174,6 +174,23 @@ This selector could have also been written as:
 }
 ```
 
+### Logical operations
+
+The `:has()` relational selector can be used to check if one of the multiple features is true or if all the features are true.
+
+By using comma-separated values inside the `:has()` relational selector, you are checking to see if any of the parameters exist. `x:has(a, b)` will style `x` if descendant `a` OR `b` exists.
+
+By chaining together multiple `:has()` relational selectors together, you are checking to see all of the parameters exist. `x:has(a):has(b)` will style `x` if descendant `a` AND `b` exist.
```
Comment (estelle) [inline files/en-us/web/css/_colon_has/index.md:183]:
```suggestion
By chaining together multiple `:has()` relational selectors together, you are checking to see if all of the parameters exist. `x:has(a):has(b)` will style `x` if descendant `a` AND `b` exist.
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**